frontend/lib/identification/card/eak_card.dart

```diff
@@ -25,94 +24,89 @@ class EakCard extends StatelessWidget {
   final BaseCardDetails cardDetails;
   final Region? region;
 
-  const EakCard({Key? key, required this.cardDetails, this.region})
-      : super(key: key);
+  const EakCard({Key? key, required this.cardDetails, this.region}) : super(key: key);
 
-  get _formattedExpirationDate {
+  String get _formattedExpirationDate {
     final expirationDate = cardDetails.expirationDate;
-    return expirationDate != null
-        ? DateFormat('dd.MM.yyyy').format(expirationDate)
-        : "unbegrenzt";
+    return expirationDate != null ? DateFormat('dd.MM.yyyy').format(expirationDate) : "unbegrenzt";
   }
 
   @override
   Widget build(BuildContext context) {
-    var cardColor =
-        cardDetails.cardType == CardType.gold ? goldenCardColor : blueCardColor;
-    return LayoutBuilder(builder: (context, constraints) {
-      final scaleFactor = constraints.maxWidth / 300;
-      final currentRegion = region;
-      final headerTitle = currentRegion != null
-          ? "${currentRegion.prefix} ${currentRegion.name}"
-          : "";
-      return Column(
+    final cardColor = cardDetails.cardType == CardType.gold ? goldenCardColor : blueCardColor;
+    return LayoutBuilder(
+      builder: (context, constraints) {
+        final scaleFactor = constraints.maxWidth / 300;
+        final currentRegion = region;
+        final headerTitle = currentRegion != null ? "${currentRegion.prefix} ${currentRegion.name}" : "";
+        return Column(
           mainAxisAlignment: MainAxisAlignment.start,
           crossAxisAlignment: CrossAxisAlignment.stretch,
           children: [
             // Header
             Container(
               color: const Color(0xf5f5f5ff),
               child: AspectRatio(
-                  aspectRatio: 6 / 1,
-                  child: Row(
-                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                      children: [
-                        EakCardHeaderLogo(
-                            title: headerTitle, scaleFactor: scaleFactor),
-                        EakCardHeaderLogo(
-                          title: "Freistaat Bayern",
-                          scaleFactor: scaleFactor,
-                          logo: const Image(
-                              image: AssetImage("assets/wappen-bavaria.png"),
-                              fit: BoxFit.contain),
-                        )
-                      ])),
+                aspectRatio: 6 / 1,
```

Comment:
No changes from me here, just applied correct formatting. AFAIK there is no option to change this.

Reply:
The previous linting setup had 4 spaces for this use case. I did run the formatter earlier already. So all files already were formatted before this branch was forked.

Reply:
Okay so I investigates why this happens and it is because of the trailing commas/where the closing brackets are placed: https://github.com/digitalfabrik/ehrenamtskarte/pull/431/commits/27f144e70fef8f611b62c5a1cd901ee84eb992b0#diff-b2fe69abdd5b15dfac1487fed9af498543f5a23894799ff3bfab54d1db899f9aR54
Without a trailing comma and brackets in the same line (`]))`) it uses 4 spaces. So this is definitely a format issue. Not sure what one should do about that here.

Reply:
E.g. in line 68 it still uses 4 spaces, so it is mixed. Same as before (also sometimes 2 and sometimes 4 spaces).